rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1,2 +1 @@
 org.jboss.resteasy.client.jaxrs.internal.ResteasyClientBuilderImpl
```

Comment:
Perhaps as the resteasy-client Jar is removed this classname should not be here as I think it may come from the jar being removed? https://github.com/resteasy/Resteasy/blob/main/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/ResteasyClientBuilderImpl.java